src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -405,6 +405,7 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSItems.ONION).grist(IODINE, 3).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.IRRADIATED_STEAK).grist(IODINE, 12).grist(URANIUM, 1).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.ROCK_COOKIE).grist(BUILD, 10).grist(MARBLE, 5).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.WOODEN_CARROT).grist(BUILD, 2).grist(ARTIFACT, 1).build(recipeSaver);
```

Comment:
I disagree with the use of artifact grist here. Sure, it's less than useful, but I don't think it should classify as shitty in the way usually related to artifact grist. (also you can't _normally_ find artifact grist currently, making the grist cost less suited for that reason as well)